src/PressureUnit.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * PressureUnit.js - ES6 wrappers around an ilib class
+ *
+ * @license
+ * Copyright © 2020, JEDLSoft
```

Comment:
2020?

Reply:
That's when I wrote it, but it was not published until now. I'll update it

Reply:
Oh no wait, this is the es6 project, not ilib itself. I copied that file originally from "LengthUnit.js" which was done in 2020